client/app/landingPage/page.jsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { Box, Paper, Stack, Typography } from "@mui/material/";
+import SignInCard from "@/components/signInCard";
+import AutoFixHighRoundedIcon from "@mui/icons-material/AutoFixHighRounded";
+import ConstructionRoundedIcon from "@mui/icons-material/ConstructionRounded";
+import ThumbUpAltRoundedIcon from "@mui/icons-material/ThumbUpAltRounded";
+
+
+//left side page content
+const items = [
+  {
+    icon: <ConstructionRoundedIcon sx={{ color: "#1565c0" }} />,
```

Comment:
This is a MUI anti pattern. You pretty much never want to use explicit colours. You want to go through the theme palette (if you aren't making use of the theme, MUI loses lots of its value imho). So something like this would be normal
```suggestion
    icon: <ConstructionRoundedIcon sx={{ color: theme => theme.palette.primary }} />,
```

For us, a better approach would probably be to define a `brand` colour or similar on our palette, since that seems to be how you guys are trying to use the yellow. So you end up with `theme => theme.palette.brand`.

I can give a hand setting this up if you fancy.